mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -4717,6 +4697,12 @@ static LogicalResult commonVerifierPackAndUnPackOp(OpTy packOrUnPack) {
     return op->emitError("mismatch in inner tile sizes specified and shaped of "
                          "tiled dimension in the packed type");
   }
+  if (failed(verifyCompatibleShape(expectedPackedType.getShape(),
+                                   packedType.getShape()))) {
+    return op->emitError("expected ")
+           << expectedPackedType << " for the unpacked domain value, got "
```

Comment:
nit: I think it should be `packed domain` - result for `pack`, input for `unpack`

Reply:
good catch, thanks!